### DISCUSSION TITLE
Does the build command generate all pages?

### DISCUSSION THREAD

**Discussion by Wysardry** [Cecilapp/Cecil#1725] [Q&A]
I checked the documentation, but couldn't find any information on whether the build command generates all the pages in a site or only those that have been changed or newly added.

I did see an option to build a single page, but that would also mean telling Cecil to update several index pages in a blog.

**Comment by ArnaudLigny** [comment]
Hello @Wysardry, the `build` command delete and recreate all the pages of a site.
